tox.ini

```
@@ -0,0 +1,75 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, static, unit
+
+[vars]
+src_path = {toxinidir}/src/
+tst_path = {toxinidir}/tests/
+lib_path = {toxinidir}/lib/charms/magma_orc8r_libs/
+all_path = {[vars]src_path} {[vars]tst_path} {[vars]lib_path}
+
+[testenv]
+basepython = python3
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+    PYTHONBREAKPOINT=ipdb.set_trace
+passenv =
+    PYTHONPATH
+    HOME
+    PATH
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
+    HTTP_PROXY
+    HTTPS_PROXY
+    NO_PROXY
+
+[testenv:fmt]
+description = Apply coding style standards to code
+deps =
+    black
+    isort
+commands =
+    isort {[vars]all_path}
+    black {[vars]all_path}
+
+[testenv:lint]
+description = Check code against coding style standards
+deps =
+    black
+    flake8
+    flake8-docstrings
+    flake8-copyright
+    flake8-builtins
+    pyproject-flake8
+    pep8-naming
+    isort
+commands =
+    # pflake8 wrapper suppports config from pyproject.toml
+    pflake8 {[vars]all_path}
+    isort --check-only --diff {[vars]all_path}
+    black --check --diff {[vars]all_path}
+
+[testenv:static]
+description = Run static analysis checks
+deps =
+    -r{toxinidir}/requirements.txt
+    mypy
+    types-PyYAML
+    pytest
+    types-setuptools
+    types-toml
+commands =
```

This will fix static analysis with latest version of mypy.
```suggestion
    types-toml
setenv =
    PYTHONPATH = ""
commands =
```